doc/portfile.7

```diff
@@ -1054,6 +1054,34 @@ to
 .br
 .Sy Example:
 .Dl use_7z yes
+.It Ic use_dmg
+Use a disk image (dmg).
+.br
+Sets
+.Ic extract.suffix
+to
+.Dq .dmg .
+.br
+Sets
+.Ic extract.cmd
+to
+.Dq hdiutil .
+.br
+Sets
+.Ic extract.pre_args
+to
+.Dq attach .
+.br
+Sets
+.Ic extract.post_args
+to
+.Dq -private -readonly -nobrowse -mountpoint ...etc... .
```

Comment:
Please refer to PR description, for further notes/comments on `extract.post_args`.